drake/CMakeLists.txt

```diff
@@ -143,6 +143,11 @@ endif()
 
 enable_testing()
 include(CTest)
+find_package(GTest REQUIRED)
+include_directories(${GTEST_INCLUDE_DIRS})
+if (GTEST_FOUND)
+  set (CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -pthread")
```

Comment:
I think this flag is no longer necessary post 8b11750, so I've removed it in 334f89e.